src/resources/js/script.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file This file contains all JavaScript specific to Schedule Day View.
+ * This file should load after all vendor and core events JavaScript.
+ * @version 1.0.0
+ */
+
+(function ( window, document, $, td, te, tf, ts, tt, config, dbug ) {
+
+	/*
+	 * $    = jQuery
+	 * td   = tribe_ev.data
+	 * te   = tribe_ev.events
+	 * tf   = tribe_ev.fn
+	 * ts   = tribe_ev.state
+	 * tt   = tribe_ev.tests
+	 * dbug = tribe_debug
+	 */
+
+	// @ifdef DEBUG
+	if ( dbug ) {
```

Comment:
We could  ditch this empty if(){} block

Reply:
agreed. i think i deleted and re-added before committing about 3 times during development :)